src/Viewer.tsx

```diff
@@ -301,9 +284,9 @@ function Viewer(): ReactElement {
       // TODO: This is a patch to keep vector state saved to the URL while the
       // state store is being refactored. This should be removed once
       // ViewerConfig is moved into the state store.
-      config: { ...config, vectorConfig },
+      // config: { ...config, vectorConfig },
       selectedBackdropKey,
-      scatterPlotConfig,
+      // scatterPlotConfig,
```

Comment:
These are commented out until I have a chance to refactor URL sync, which should be the next PR after this one.